src/components/subtitlesettings/subtitlesettings.scss

```diff
@@ -24,3 +24,42 @@
     display: inline-block;
     max-width: 70%;
 }
+
+#subtiteColorsContainer {
+    margin: .2em 0;
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+    gap: .1em;
+}
+
+input[name=subcolor] {
+    display: none;
+}
```

Comment:
fixed... sort of
Keyboard and remote can be used now but experience is not very great.
Left-right keys select the colors while up and down allow to enter, traverse and exit the area.
It is not very intuitive. With just one row it would be probably ok.
With multiple rows perhaps best would be to do grid with navigation using arrow keys and selecting the color by pressing center button/enter/space.

Reply:
> Left-right keys select the colors while up and down allow to enter, traverse and exit the area.

#4330

For testing purposes, remove `INPUT` here:
https://github.com/jellyfin/jellyfin-web/blob/c4979dcce43ce8b8d0afa4252129c0d9ea2feb82/src/scripts/keyboardNavigation.js#L51